<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="lbl_libros">
            <property name="text">
             <string>Libros</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListView" name="lv_libros"/>
          </item>
          <item>
           <widget class="QPushButton" name="pb_crearLibro">
            <property name="text">
             <string>Crear Libro</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_eliminarLibro">
            <property name="text">
             <string>Eliminar Libro</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="f_secciones">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="lbl_secciones">
            <property name="text">
             <string>Secciones</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListView" name="lv_secciones"/>
          </item>
          <item>
           <widget class="QPushButton" name="pb_crearSeccion">
            <property name="text">
             <string>Crear Sección</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_eliminarSeccion">
            <property name="text">
             <string>Eliminar Sección</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="f_paginas">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="lbl_paginas">
            <property name="text">
             <string>Páginas</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListView" name="lv_paginas"/>
          </item>
          <item>
           <widget class="QPushButton" name="pb_crearPagina">
            <property name="text">
             <string>Crear Página</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pb_eliminarPagina">
            <property name="text">
             <string>Eliminar Página</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="f_notass">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="lbl_notass">
            <property name="text">
             <string>Notas</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListView" name="lv_notass"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Crear Nota</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Eliminar Nota</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="q_frameNota">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QFrame" name="f_nota">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="lbl_nota">
            <property name="text">
             <string>Nota</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="txt_nota"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_archivar">
         <property name="text">
          <string>Archivar Nota</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
